packages/p2p-v2/src/components/LightDivider/LightDivider.tsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import { Divider } from '@deriv-com/ui';
+
+type TLightDividerProps = {
+    className?: string;
+    height?: string;
+    margin?: string;
+};
+
+const LightDivider = ({ className, height, margin }: TLightDividerProps) => {
+    return <Divider className={className} color='#f2f3f4' height={height} margin={margin} />;
```

Comment:
```suggestion
    return <Divider color='#f2f3f4' {...props} />;
```

Reply:
Hmmm i think having the props defined explicitly is better since there aren't as many props that are needed to be passed to this small component, if there were a lot more props it would be better to use the spread operator... plus we don't want the colour prop to be overriden in this case, since this type of divider was created because of the colour being used in a lot of places